src/ucp/api/ucp.h

```diff
@@ -333,6 +335,19 @@ typedef enum {
 
 /**
  * @ingroup UCP_DATATYPE
+ * @brief Generate a reusable identifier for Scatter-gather IOV data type.
```

Comment:
1. Why does this change justify changing all the code written so far?
2. How is this relate to "reusability"? how does this API allow you to state whether the memh can be cached?